ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -998,6 +1053,13 @@ for target in "${TARGET_RELEASES[@]}"; do
     fi
     check_history
     health_check
+    currentVersion=$(oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1)
+    currentPlugin=$(oc get network.config.openshift.io cluster -o jsonpath='{.status.networkType}')
+    if [[ ${currentVersion} == "4.16" && ${currentPlugin} == "OpenShiftSDN" ]]; then
```

Comment:
here we can check TARGET_MINOR_VERSION == "16"

Reply:
updated, thanks